src/lib/datamodel/ClusterOnOff.h

```diff
@@ -39,12 +39,12 @@ static const uint16_t kAttributeIdOnTime             = 0x4001;
 static const uint16_t kAttributeIdOffWaitTime        = 0x4002;
 
 /* Command IDs */
-static const uint16_t kOnOffCmdIdOff                  = 0x00;
-static const uint16_t kOnOffCmdIdOn                   = 0x01;
-static const uint16_t kOnOffCmdIdToggle               = 0x02;
-static const uint16_t kOnOffCmdIdOffWithEffect        = 0x40;
-static const uint16_t kOnOffCmdIdOffWithRecall        = 0x41;
-static const uint16_t kOnOffCmdIdOnWithTimedOff       = 0x42;
+static const uint16_t kOnOffCmdIdOff            = 0x00;
```

Comment:
cmdid should be a type, maybe?

Reply:
Sure, also for the others (clusterID, attrID), will update those in one of the commits

Reply:
done